report/report.tex

```diff
@@ -348,6 +361,14 @@ \section{Reflection}
         Although everybody was aware of the amount of work that had to be put into the Eval module from the get go, it still cost a lot more time eventually. For a problem that, we all already recognised back then, was going to be difficult, more time should have been claimed since a problem that seems theoretically challenging is almost always far more difficult in practice. However, a mistake was also made by not correctly working in a strictly iterative fashion. The result of this was that the Eval module was almost always broken in one way or another because it was still not finished. Working in a strictly iterative fashion, always working on one function and one function only, would have resulted in a clearer overview of what the tasks and goals were at every moment, and in clearer moments on which the only partially finished, but stable, Eval module could be merged with the master branch.
         
 Due to an initial lack of tests for all possible Hanus constructs, progress on the evaluator was slightly delayed towards the end of the project, since a few previously unknown bugs in the parser became exposed at that time. 
+\section{Future work}
+	The Hanus project can be extended or improved in multiple ways. Some ideas for future work on Hanus are:
+        \begin{enumerate}
+                	\item Implementing proper field and array indexers, i.e. at the moment, multiple concatenated field and array indexers are not supported.
```

Comment:
Is 'concatenated' the right word? "A sequence of multiple field and array indexers is not supported"?